crates/autopilot/src/driver_model.rs

```diff
@@ -109,8 +109,8 @@ pub mod solve {
         pub sell_token_balance: SellTokenSource,
         pub buy_token_balance: BuyTokenDestination,
         pub class: Class,
-        #[serde_as(as = "Option<DecimalU256>")]
-        pub surplus_fee: Option<U256>,
+        //#[serde_as(as = "Option<DecimalU256>")] todo cleanup
+        //pub surplus_fee: Option<U256>,
```

Comment:
If you can comment it out then why can't you delete it?

Reply:
Wanted to have it as a reminder for the follow up cleanup PR, but I can remove it now also.